gateway/1.6/modules/ROOT/pages/_partials/task-config-disable-logs.adoc

```diff
@@ -0,0 +1,147 @@
+//tag::config-disable-logs-intro[]
+
+Anypoint Flex Gateway logging provides insights into startup, shutdown, API requests, API responses, policies, and Fluent Bit. Logging allows developers and administrators to identify and address issues or anomalies. However, disabling Anypoint Platform log output might become necessary. For example, regulatory or compliance frameworks can impose restrictions on retaining logs beyond a specific period. Disabling logs ensures compliance with such requirements.
+
```

Comment:
@seblaz I’ll remove the mention of compliance, because based on your comments it doesn’t sound like I had described a valid reason to disable sending logs to Anypoint Platform. I think we can assume that the customer will already know the reasons if their deployment requires disabling these logs.